ynput/OpenPype
# Settings: Houdini & Maya create plugin settings

## Changelog Description
Fixes related to Maya and Houdini settings. Renamed `defaults` to `default_variants` in plugin settings to match attribute name on create plugin in both OpenPype and AYON settings. Fixed Houdini AYON settings where were missing settings for defautlt varaints and fixed Maya AYON settings where default factory had wrong assignment.

## Additional information
Added `"Main"` back to default settings so manager who change the settings can see what the default actually is.

## Testing notes:
Settings for Maya and Houdini settings should be applied to creators
1. Change default variants in a creator plugin settings
2. Open the host where changes happened
3. Select creator
4. Validate that default variant and hints are matching settings

Houdini AYON addon should not cause crash on server restart.
